src/plot_api/plot_api.js

```diff
@@ -196,8 +196,8 @@ Plotly.plot = function(gd, data, layout, config) {
 
         Registry.getComponentMethod('legend', 'draw')(gd);
         Registry.getComponentMethod('rangeselector', 'draw')(gd);
-        Registry.getComponentMethod('updatemenus', 'draw')(gd);
         Registry.getComponentMethod('sliders', 'draw')(gd);
+        Registry.getComponentMethod('updatemenus', 'draw')(gd);
```

Comment:
Real simple. The fact that it's quite this simple is a little concerning though.

- [Updatemenu is drawn on `_infolayer`](https://github.com/plotly/plotly.js/blob/master/src/components/updatemenus/draw.js#L55)
- [Slider is drawn on `_infolayer`](https://github.com/plotly/plotly.js/blob/master/src/components/sliders/draw.js#L29)

I guess that means it works. For some reason I thought the dropdown was drawn on a separate layer (should it be?). The only corner case I can think of:

1. Draw plot with updatemenus and no sliders
2. Add slider
3. Somehow the slider is appended *after* the updatemenu and ends up on top of it

But I guess that's what containerClass is for. So perhaps this is adequate.